### PR TITLE
modify chrome extension url

### PR DESCRIPTION
**Pull request by ysm0622**
In the chrome extension, you can temporarily switch between production and development via variable `isProduction(boolean)`toggle.

### CODE REVIEW

**Comment by gimdongwoo** [comment]
How to change automatically `isProduction`?

**Comment by gimdongwoo** [comment]
How do you think about this?
https://stackoverflow.com/questions/12830649/check-if-chrome-extension-installed-in-unpacked-mode

**Comment by gimdongwoo** [comment]
```
function isDevMode() {
    return !('update_url' in chrome.runtime.getManifest());
}
```

File: chrome-extension/src/index.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
 let editor;
 let iframe;
+const isProduction = true;
```
Comment (gimdongwoo) [inline chrome-extension/src/index.js]: function isDevMode() {
    return !('update_url' in chrome.runtime.getManifest());
}

**Comment by ysm0622** [comment]
Edited with your suggestion.

**Comment by ysm0622** [comment]
Edited with your suggestion.

File: chrome-extension/src/index.js [hunk 2]
```diff
@@ -124,3 +119,9 @@ if (!window.location.ancestorOrigins.contains(extensionOrigin) && !document.getE
     }
   });
 }
+
+window.addEventListener('click', e => {
+  if (document.querySelector('._1fng') && !isHaveBtnAleady()) {
```
Comment (jayjongcheolpark) [inline chrome-extension/src/index.js]: @ysm0622 `._1fng` does not look like a meaningful name. Everything else is good 👍

Comment (ysm0622) [reply chrome-extension/src/index.js]: `_1fng` is FB's specific component hashed class name.

**Pull request merged**